client/components/notice/test/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { Notice } from '../index';
+
+describe( 'index', function() {
+	it( 'should output the component', function() {
+		const wrapper = shallow( <Notice translate={ identity } /> );
+		assert.equal( true, wrapper.find( '.notice' ).length );
```

Comment:
`length` is equal to `true` ? I guess `assert.equal` isn't doing strict equality. 

You might want this instead: http://chaijs.com/api/assert/#method_strictequal

Or for general truthiness: http://chaijs.com/api/assert/#method_isok

Or testing the length: http://chaijs.com/api/assert/#method_lengthof